mmv1/products/migrationcenter/Group.yaml

```diff
@@ -0,0 +1,87 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+base_url: projects/{{project}}/locations/{{location}}/groups
+create_url: projects/{{project}}/locations/{{location}}/groups?groupId={{group_id}}
+self_link: projects/{{project}}/locations/{{location}}/groups/{{group_id}}
+id_format: projects/{{project}}/locations/{{location}}/groups/{{group_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/groups/{{group_id}}
+name: Group
+description: Description
+update_verb: :PATCH
+update_mask: true
+autogen_async: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'migration_group_basic'
+    primary_resource_id: 'default'
+    vars:
+      group_name: 'group-test'
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    description: 'Output only. The name of the group. '
+    output: true
+  - !ruby/object:Api::Type::String
+    name: createTime
+    description: 'Output only. The timestamp when the group was created. '
+    output: true
+  - !ruby/object:Api::Type::String
+    name: updateTime
+    description: 'Output only. The timestamp when the group was last updated. '
+    output: true
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: labels
+    description: 'Labels as key value pairs. '
+  - !ruby/object:Api::Type::String
+    name: displayName
+    description: 'Optional. User-friendly display name. '
+  - !ruby/object:Api::Type::String
+    name: description
+    description: 'Optional. The description of the group. '
+parameters:
+  - !ruby/object:Api::Type::String
+    name: location
+    description: 'Part of `parent`. See documentation of `projectsId`. '
```

Comment:
Nit: I'm guessing this was copied, but I think it is a little confusing. I didn't see any mention of `parent` or `projectsId` in this resource, so I'm not sure how a user would be able to make use of this information.

Reply:
Yeah, it's a generated resource so it's not very helpful. I'll fix this manually